### PR TITLE
[MTB] Bugs removal and ux enhancement

### PR DESCRIPTION
**Pull request by phoenixking25**
/assign @JimBugwadia

### CODE REVIEW

**Comment by phoenixking25** [comment]
fix https://github.com/kubernetes-sigs/multi-tenancy/issues/943, https://github.com/kubernetes-sigs/multi-tenancy/issues/941

File: benchmarks/kubectl-mtb/internal/kubectl-mtb/get.go [hunk 1]
```diff
@@ -27,9 +27,22 @@ import (
 )
 
 var getCmd = &cobra.Command{
-	Use:   "get",
+	Use:   "get <resource>",
 	Short: "Display one or many benchmarks.",
 
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return fmt.Errorf("Please specify any resource")
```
Comment (JimBugwadia) [inline benchmarks/kubectl-mtb/internal/kubectl-mtb/get.go:35]: Change to `Please specify a resource`.

File: benchmarks/kubectl-mtb/internal/kubectl-mtb/get.go [hunk 2]
```diff
@@ -27,9 +27,22 @@ import (
...
+		}
+		if !supportedResourceNames.Has(args[0]) {
+			return fmt.Errorf("Please specify any valid resource")
```
Comment (JimBugwadia) [inline benchmarks/kubectl-mtb/internal/kubectl-mtb/get.go:38]: Change to `Please specify a valid resource`.

Assuming we are showing the user the list of valid resources, right?

File: benchmarks/kubectl-mtb/internal/kubectl-mtb/run.go [hunk 3]
```diff
@@ -37,12 +37,28 @@ var (
...
 
-var testCmd = &cobra.Command{
-	Use:   "test",
+var runCmd = &cobra.Command{
+	Use:   "run <resource>",
 	Short: "Run the Multi-Tenancy Benchmarks",
 
+	PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
+		if len(args) == 0 {
+			return fmt.Errorf("Please specify any resource")
```
Comment (JimBugwadia) [inline benchmarks/kubectl-mtb/internal/kubectl-mtb/run.go:46]: Same comments as above

**Pull request merged**